modules/nodes-pods-secrets-about.adoc

```diff
@@ -64,10 +64,11 @@ which is the default.
 
 Specify one of the following types to trigger minimal server-side validation to ensure the presence of specific key names in the secret data:
 
-* `kubernetes.io/service-account-token`. Uses a service account token.
-* `kubernetes.io/basic-auth`. Use with Basic Authentication.
-* `kubernetes.io/ssh-auth`. Use with SSH Key Authentication.
-* `kubernetes.io/tls`. Use with TLS certificate authorities.
+* `kubernetes.io/basic-auth`: Use with Basic authentication
+* `kubernetes.io/dockercfg`: Use as an image pull secret
+* `kubernetes.io/service-account-token`: Uses a service account token
```

Comment:
Would you like to add both `kubernetes.io/dockercfg` and `kubernetes.io/dockerconfigjson`? This doc's other places mention `kubernetes.io/dockerconfigjson` examples. Upstream https://kubernetes.io/docs/concepts/configuration/secret/#secret-types has examples of both.

Reply:
We definitely could. @sanchezl Do you know what we could put as the description for the dockerconfigjson one? Would it still be used for an image pull secret, but just a different file (`~/.docker/config.json`), or what?

Reply:
@bergerhoffer The description would be exactly the same. The format of the file/data would be slightly different in the actual secret.

Reply:
Thanks! I updated to add it with the same description. @xingxingxia @sanchezl if you can take another look and let me know if it's okay the way it is, or if the description needs adjusted to be different between the two.